src/key_io.cpp

```diff
@@ -286,7 +286,11 @@ libzcash::PaymentAddress DecodePaymentAddress(const std::string& str)
     }
     data.clear();
     auto bech = bech32::Decode(str);
-    if (bech.first == Params().Bech32HRP(CChainParams::SAPLING_PAYMENT_ADDRESS) &&
+    bool allowSapling = Params().NetworkIDString() == "regtest" || (
```

Comment:
Make a global function for this since repeated in several places?

Reply:
Not worth the overhead, as we'll only be using this for a few weeks, and then will strip it out completely.